### DISCUSSION TITLE
Battery Level?

### DISCUSSION THREAD

**Discussion by sstratoti** [RobertD502/home-assistant-flair#39] [Q&A] [answered]
Hi Robert! Not sure if this is possible but thought I'd ask. I'm getting those "low battery" emails from flair, but sometimes I miss them. In HA, I have an auto entities which displays batteries under a certain percentage on my main view. It's kind of a "hey, you should change these" type of indicator. But I only see voltage coming through with your integration. Is there any way to determine the battery percentage? Or should I be looking for voltage under 3V or something as an indicator?

Thanks!

**Comment by RobertD502** [comment, marked as answer]
Hi Steve! So, Flair doesn't report the battery in %...only Voltage, which you've already gathered.

I would take a look at [this Flair support article.](https://support.flair.co/hc/en-us/articles/360044126272-Low-Battery-Alert) As you can see, depending on what device you're talking about (puck vs vent), the normal range differs. 

Where it gets interesting is that the wording of the support article doesn't make it clear if the 2.4V measurement is the point where a low battery notification is sent for both Pucks and Vents or just Vents - there wouldn't be any confusion if the Pucks and Vents were said to have the same normal range. My guess is alerts are sent out at 2.7V for pucks (0.1 below normal range) and 2.4V for vents (0.1 below normal range). 

The reason I didn't incorporate percentages is because I'm not 100% (pun intended) sure what Flair views 2.4V as - is it 5% battery left, is it 10% battery left, etc. I'll try to reach out to them and see if they have data indicating voltage to battery percentage conversions - I'm not so confident about this since their own support article is telling users to look at voltage readings.

My suggestion is to use "less than or equal to 2.4V" for vents to create alerts in Home Assistant. I don't power any pucks with batteries so I can't test if 2.7V is when Flair sends alerts for pucks.

**Reply by sstratoti** [reply]
Thank you! This is what I was thinking. Appreciate all the work you do!
